tests/Browser/CreateTaskCategoryValidationTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Tests\Browser;
+
+use Tests\DuskTestCase;
+use Laravel\Dusk\Browser;
+use App\Models\TaskCategory;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+
+class CreateTaskCategoryValidationTest extends DuskTestCase
```

Comment:
No need to create a separate class for testing negative cases. All test cases related to Task Creation should go in a single class i.e. in `CreateTaskCategoryTest` class.

Reply:
Do I have to use if condition ?